client/packages/invoices/src/InboundShipment/DetailView/AddButton.tsx

```diff
@@ -67,7 +68,7 @@ export const AddButton = ({
   const handleOptionSelection = (option: SplitButtonOption<string>) => {
     switch (option.value) {
       case 'add-item':
-        onAddItem(true);
+        onAddItem();
```

Comment:
The `true` wasn't necessary here -- the function opens the modal either way, so we can now use this input parameter for the scanned data.